crate/cli/src/actions/permissions.rs

```diff
@@ -70,6 +73,37 @@ impl CreateIndex {
     }
 }
 
+/// List user's permission. Returns a list of indexes with their permissions.
+#[derive(Parser, Debug)]
+pub struct ListPermissions {
+    /// The user identifier to allow
+    #[clap(long, required = true)]
+    pub user: String,
```

Comment:
Can a user list the permissions of another user ? Are the users discoverable?

Imagine I am an attacker and stole some credentials that only have a read permission: can I identify which user has write permissions?

Reply:
Currently, knowing users id, an authenticated attacker can list users permissions. He won't be able to appropriate itself users permissions unless he stole a JWT access token or a PKCS12 certificate.